Makefile

```diff
@@ -16,12 +16,13 @@ MCLI_E2E_BINARY?=../../bin/${MCLI_BINARY_NAME}
 
 ATLAS_SOURCE_FILES?=./cmd/atlas
 ATLAS_BINARY_NAME=atlas
+ATLAS_VERSION?=$(shell git tag --list 'atlascli/v*' --sort=committerdate | tail -1 | cut -d "v" -f 2 | xargs -I % sh -c 'echo %-next' )
```

Comment:
This piece of code and the evergreen packaging tasks won’t work until we have an `atlascli/v*` tag in the repository